### PR TITLE
fix:[PAYM-2296] Fixed breaking components

### PR DESCRIPTION
**Pull request by MekhrangizY**
**PR description:**

**Implemented:**

**Checklist:**

- [x] I ran this code locally
- [ ] I wrote the necessary tests
- [x] My code follows the [style guidelines](http://bit.ly/sd-web-style-guide)
- [x] I followed the [instructions](http://bit.ly/sd-web-pr) to create a pull request

**JIRA card:**

**Should know about:**

### CODE REVIEW

File: packages/ui/src/theme/Color.ts [hunk 1]
```diff
@@ -336,7 +179,8 @@ export enum ColorDarkInternal {
   // ???
   //
 
-  White = '#fff',
+  White = 'rgba(255, 255, 255, 0.5)',
+  DynamicWhite = 'rgba(25, 35, 52, 1)',
```
Comment (timurhamitov) [inline packages/ui/src/theme/Color.ts:183]: @MekhrangizY **nitpick**: if it's do not have opacity, maybe we can use hex value instead?

**Review by timurhamitov — COMMENTED** [review]
@MekhrangizY Have you showed to Artyom how components looks like after changes?

File: packages/ui/src/theme/Color.ts (hunk 1 above)
Comment (MekhrangizY) [reply packages/ui/src/theme/Color.ts:183]: Artyom told to put it in rgba -  DynamicWhite = 'rgba(25, 35, 52, 1)',
for white sure i will change to hex

**Review by timurhamitov — APPROVED** [review]

File: packages/lab/src/box/Box.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ import { createRuleNormalizer, RuleNormalizer } from '../utils/RuleNormalizer';
 //
 
 function normalizeColor(input: unknown): string | undefined {
-  return isColorPropV2(input) ? ColorV2[input] : undefined;
+  return isColorPropV2(input) ? ColorDynamic[input] : undefined;
```
Comment (doniyor2109) [inline packages/lab/src/box/Box.tsx:21]: `isColorPropV2` -> `isColorDynamicProp`

File: packages/ui/src/__tests__/index.spec.ts [hunk 3]
```diff
@@ -109,7 +109,7 @@ it('exposes public api', () => {
         "Dark400": "#323C4E",
         "Dark450": "#222F44",
         "Dark50": "rgba(25, 35, 52, 0.5)",
-        "Dark500": "#FFFFFF",
+        "Dark500": "#fff",
```
Comment (doniyor2109) [inline packages/ui/src/__tests__/index.spec.ts:112]:
```suggestion
        "Dark500": "#FFFFFF",
```

File: packages/ui/src/theme/Color.ts [hunk 4]
```diff
@@ -640,7 +482,7 @@ export enum ColorV2 {
 }
 
 export type ColorProp = keyof typeof Color;
-export type ColorPropV2 = keyof typeof ColorV2;
+export type ColorPropV2 = keyof typeof ColorDynamic;
```
Comment (doniyor2109) [inline packages/ui/src/theme/Color.ts:485]:
```suggestion
export type ColorDynamicProp = keyof typeof ColorDynamic;
```

File: packages/ui/src/theme/Color.ts [hunk 5]
```diff
@@ -453,7 +295,7 @@ export enum ColorDarkInternal {
   Dark50 = 'rgba(25, 35, 52, 0.5)',
   Dark100 = '#6E7A8A',
   Dark300 = '#AEB4BD',
-  Dark500 = '#FFFFFF',
+  Dark500 = '#fff',
```
Comment (doniyor2109) [inline packages/ui/src/theme/Color.ts:298]:
```suggestion
  Dark500 = '#FFFFFF',
```

File: packages/ui/src/theme/Color.ts (hunk 4 above)
Comment (MekhrangizY) [reply packages/ui/src/theme/Color.ts]: done

File: packages/ui/src/theme/Color.ts [hunk 6]
```diff
@@ -640,7 +482,7 @@ export enum ColorV2 {
 }
 
 export type ColorProp = keyof typeof Color;
```
[the remark this answers is not in the document]
Comment (MekhrangizY) [reply packages/ui/src/theme/Color.ts:484]: done

File: packages/ui/src/theme/Color.ts (hunk 5 above)
Comment (MekhrangizY) [reply packages/ui/src/theme/Color.ts]: done

File: packages/lab/src/box/Box.tsx (hunk 2 above)
Comment (MekhrangizY) [reply packages/lab/src/box/Box.tsx]: done

**Pull request merged**